app/controllers/api/v1/base_controller.rb

```diff
@@ -1,10 +1,12 @@
-class Api::V1::BaseController < ApplicationController
+class Api::V1::BaseController < ActionController::API
```

Comment:
So now the api controller and regular controllers do not have single parent ? //cc @vipulnsward @prathamesh-sonpatki 

You have moved common stuff like "setting honeybadger" to a module and that module is being included in both the controllers. However now `BasicUserSetting` has "layout carrier" which is not needed by api controllers. Hence I prefer having common stuff at "ApplicationController" rather than into a module.